src/Workspaces/CSharp/Portable/Utilities/CSharpTypeStyleHelper.State.cs

```diff
@@ -14,7 +14,7 @@ namespace Microsoft.CodeAnalysis.CSharp.Utilities
 {
     internal partial class CSharpTypeStyleHelper
     {
-        protected class State
+        protected struct State
```

Comment:
An instance of this is produced for every node we want ot try to simplify/expand 'var' for.  No need to allocate here.